Automattic/jetpack
# CLI: Add Squash to jetpack changelog

#### Changes proposed in this Pull Request:

Adds the squash command to the jetpack changelogger tool.*

#### Jetpack product discussion


N/a
#### Does this pull request change what data or activity we track or use?


No
#### Testing instructions:





Unsure how to test without having a few alphas to go off of. Could be good to merge and then I can test later in the month after a few alphas are in.

Should be run with `jetpack changelog squash [project] <readme>`

This lets you specify if you just want to run the squash command, or just squash the readme, since that's what we need for releases. 

Try `jetpack changelog squash plugins/jetpack -r` to run the process currently outlined in the release documentation.
